keps/sig-storage/1472-storage-capacity-tracking/README.md

```diff
@@ -795,79 +840,149 @@ enhancement:
+
+If for whatever reason the first phase fails, the feature can be left
+disabled and the second phase can be skipped. The cluster the operates
+as before.
+
+If the second phase fails because a driver malfunctions or overloads
+the apiserver, then it can be rolled back and scheduling again happens
+without using storage capacity information.
+
+In none of these cases are running workloads affected.
+
 * **What specific metrics should inform a rollback?**
 
+Increased number of pods that are not getting scheduled and apiserver
+load.
+
 * **Were upgrade and rollback tested? Was upgrade->downgrade->upgrade path tested?**
 
+No.
+
 * **Is the rollout accompanied by any deprecations and/or removals of features,
   APIs, fields of API types, flags, etc.?**
 
-### Monitoring requirements
+No.
 
-Will be added before the transition to beta.
+### Monitoring requirements
 
 * **How can an operator determine if the feature is in use by workloads?**
 
+The feature is used if all of the following points apply:
+- `CSIDriver.spec.storageCapacity` is true for a CSI driver.
+- There are storage classes for that driver.
+- Volumes are using those storage classes.
+
 * **What are the SLIs (Service Level Indicators) an operator can use to
   determine the health of the service?**
 
+Pod status of the CSI driver deployment, existence of
+`CSIStorageCapacity` objects and ultimately the rate at which volumes
+are being created by the driver. The latter is exposed through the
+existing CSI sidecar metrics support.
+
 * **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
 
+This depends on the workload and CSI driver. A very low rate of
+provisioned volumes per second and few total number of volumes overall
+may be fine for some workloads (long-running applications which use
+specialize storage like PMEM) while others may need a much higher rate
+(adding a local LVM scratch volume to every pod in the system).
+
 * **Are there any missing metrics that would be useful to have to improve
   observability if this feature?**
 
-### Dependencies
+No.
 
-Will be added before the transition to beta.
+### Dependencies
 
 * **Does this feature depend on any specific services running in the cluster?**
 
-### Scalability
+No, just the ones that will also run without it enabled (apiserver,
+kube-scheduler, CSI driver).
 
-Will be added before the transition to beta.
+### Scalability
 
 * **Will enabling / using this feature result in any new API calls?**
 
+Yes.
+
+Enabling it in apiserver and CSI drivers will cause
+`CSIStorageCapacity` objects to be created respectively updated. The
+number of those objects is proportional to the number of storage
+classes and number of distinct storage topology segments. For
+centralized provisioning, the number of segments is probably low. For
+distributed provisioning, the each node where the driver runs
+represents one segment. The rate at which objects depends on how often
+storage usage changes.
```

Comment:
It would be good to be more specific, e.g.:

- distributest provising:
  #CSIStorageCapacity per each (node, storage-class) pair  [is that right?]

For rate of changes, we can probably estimate it:
- a creation whenever node is create
- an update whenever: some volume is created/resized/deleted (i.e. bounded by volume api calls) or storage underneath is changed (which probably requires some operator action)

Reply:
Added.